added support for extNamespace and extSassNamespace

aholtkamp opened the pull request.

aholtkamp committed:
added support for extNamespace and extSassNamespace

fwienber commented on idea-plugin/src/main/java/net/jangaroo/ide/idea/JangarooFacetImporter.java:
```diff
@@ -470,10 +472,26 @@ private void configureMxmlNamespaces(MavenProject mavenProjectModel, ModifiableF
+          allOptions.put("extSassNamespace", extSassNamespace);
+        }
+      }
+      if (!allOptions.isEmpty()) {
```
This method also updates an existing buildConfiguration. This means it could happen (while unlikely) that the buildConfiguration _becomes_ empty, e.g. when you remove all corresponding elements from the Maven <configuration>. So I think you should _always_ setAllOptions(), even when allOptions is empty.

aholtkamp replied:
So we are fixing a potentional bug here? The current behavior is the same as before.

aholtkamp committed:
always set all options

in case something has switched being set to nothing being that that build configuration would not be updated otherwise
